torch/csrc/jit/passes/onnx/remove_inplace_ops_for_onnx.cpp

```diff
@@ -668,30 +680,30 @@ static void PrepareListPopForONNX(Node* n) {
     // And replace all uses of %seq afterwards with %new_seq
     Node* getitem_node =
         n->owningGraph()->create(aten::__getitem__, {n->inputs()});
-    getitem_node->output()->copyMetadata(n->output());
+    getitem_node->output()->setType(n->output()->type());
     getitem_node->insertBefore(n);
-    n->output()->replaceAllUsesWith(getitem_node->output());
```

Comment:
i see why in our discussion the output is not replaced .. this line is important if the pop item is actually used as well.